### ISSUE TITLE
virtiodrmfb: mmap failed: Invalid argument

### ISSUE THREAD

**Issue by jwilk** [jwilk/fbcat#10]
```console
$ cat /proc/fb
0 virtiodrmfb

$ fbcat > /dev/null
fbcat: mmap failed: Invalid argument
```

**Comment by jwilk** [comment]
This is the driver you get with QEMU's `-vga virtio`.

```console
$ uname -a
Linux lagrange 4.6.0-1-amd64 #1 SMP Debian 4.6.4-1 (2016-07-18) x86_64 GNU/Linux
```

**Comment by jwilk** [comment]
Fixed in 03441d7c5613e1270615168c72687460e42b2d95.

I'd still love to know why is this FB device non-`mmap()`-able.
I tried looking at the kernel source (`drivers/gpu/drm/virtio`), but it's all Greek to me. :frowning:

**Comment by jwilk** [comment]
Fixed in 0.4.4.
